test/functional/JLM_Tests/src/org/openj9/test/java/lang/management/TestRuntimeMXBean.java

```diff
@@ -619,10 +625,10 @@ public final void testGetMBeanInfo() {
 		// Print description and the class name (not necessarily identical).
 		logger.debug("MBean description for " + rb.getClass().getName() + ": " + mbi.getDescription());
 
-		// Sixteen attributes - none writable - and five IBM specific.
+		// Sixteen attributes - none writable - and eight IBM specific.
 		MBeanAttributeInfo[] attributes = mbi.getAttributes();
 		AssertJUnit.assertNotNull(attributes);
-		AssertJUnit.assertTrue(attributes.length == 22);
+		assertEquals(attributes.length, 25, "wrong number of attributes");
```

Comment:
The first argument should be the expected number.
I think a qualified reference to `assertEquals` here is preferable to a static import.